### DISCUSSION TITLE
GraphQL Support [Jul 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2944] [Discord Threads]
> 🙋‍♂️ **Eustachio**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@962537363983196240> GraphQL support is deprecated.  
#  
> 🙋‍♂️ **Eustachio**  
#### Oh! I see. So If I want to use nocodb as a cms I guess I need to get familiar with the rest API or the SDK  
#  
> 🙋‍♂️ **Eustachio**  
#### I have seen some e-commerce apps using Airtable as a cms, is it possible to use nocodb as an e-commerce backend?  
#  
> 💁 **o1lab**  
#### you can use this as a CMS  
#  
> 💁 **o1lab**  
#### can u expand in the usecase of e-commerce backend of airtable  
#  
> 🙋‍♂️ **Eustachio**  
#### More like softr.io is using Airtable for their backend  
#  
> 🙋‍♂️ **Eustachio**  
#### In my case I'd like to use nocodb as a headless CMS for an ecommerce shop. Obviously all the data will be created in nocodb, products, collections, orders, customers, etc and then being fetch by the frontend  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999838923549311008)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
